### ISSUE TITLE
Protein/DNA complex example (1lmb.pdb) would disassociate and protein would fold by itself after ~1 ns

### ISSUE THREAD

**Issue by yihengwuKP** [cabb99/open3spn2#10]
**Describe the bug**
After running the `Protein_DNA_example.py`  for 10^7 steps (timestep = 2 fs), the 1lmb protein/dna complex would fully disassociate and the protein would further fold (compared with its conformation in 1lmb.pdb, where the protein extends to fit in with the major groove). To be more specific, the recognition through two major grooves disassociate fully at even 5e5 steps (~ 1 ns) and in the same time the protein folds more compactly, after which the protein only interact with the terminal of the DNA, and at around 5e6 steps the protein and DNA lose contact. 

**To Reproduce**
1. open the `Protein_DNA_example.py` under `open3spn2/example/Protein_DNA`
2. edit the line 186 from `simulation.step(100000)` to `simulation.step(10000000)`
3. run the `Protein_DNA_example.py`

**Expected behavior**
As described in the [OpenAWSEM/Open3SPN2 paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008308#sec014), I expected OpenAWSEM and Open3SPN2 make the protein/dna complex at least fluctuate around the original pdb conformation. I know that the docking is generally a hard problem, but it seems in this case the protein/dna just diffuse away from the starting conformation, indicating the pdb conformation is probably not even a local minimum? I report this issue just to make sure there is nothing I did wrong when using openawsem/open3spn2 or this is actually expected behaviour of current OpenAWSEM/Open3SPN2 (since this is actually the example provided in the paper). 

**Screenshots**
- the start conformation
![1lmb_start](https://user-images.githubusercontent.com/39614623/130649989-96b0e4e1-0a6a-4956-87d2-05bc30547b03.png)

- the end conformation, where the protein and dna disassociate fully
![1lmb_end](https://user-images.githubusercontent.com/39614623/130650095-4489ab66-4a47-464f-a464-1d7f8c67a16a.png)

The whole trajectory can be found here: [1lmb_traj.tar.gz](https://github.com/cabb99/open3spn2/files/7040566/1lmb_traj.tar.gz)

**Desktop (please complete the following information):**
 - OS: Centos (cluster in uchicago, midway-rcc)

**Comment by cabb99** [comment]
Thank you for your feedback. Yes, as far as I know this may be an expected behavior. 

Currently the two main forces implemented for the DNA-protein interaction are a Lenard-jones repulsion potential and a electrostatic interaction. We found that this forces may be enough to simulate proteins where the protein-DNA interaction occurs between the phosphates and charged aminoacids, giving it a non-specific nature.

Here are some examples:
[https://doi.org/10.1063/1.5085663](https://doi.org/10.1063/1.5085663)
[https://pubs.acs.org/doi/full/10.1021/acs.jpcb.6b02359](https://pubs.acs.org/doi/full/10.1021/acs.jpcb.6b02359)

We chose the example in the paper as a simple system that could be set up and ran in a short amount of time, but the interactions between the lambda repressor and DNA are specific, where the protein sidechain interact with the DNA bases, and we are currently working on implementing the specific interactions.
